src/app/_components/ArticleHeader.tsx

```diff
@@ -20,17 +21,15 @@ export function ArticleHeader({ image, children }: ArticleHeaderProps) {
   return (
     <header className="space-y-6">
       <div className="space-y-6">{children}</div>
-
       <div className="relative aspect-video">
-        <DynamicImage
+        <ImageWithFallback
```

Comment:
Why do we need to name it explicitly `WithFallback`? I kinda expect a fallback out of the box?